Makefile

```diff
@@ -30,7 +36,7 @@ TEST_OBJS = $(filter-out build/obj/main.o,$(OBJS))
 TESTER = valgrind -q
 
 # All the make rules
-.PHONY: all clean install test FORCE_RULE
+.PHONY: all clean install test style style_clean FORCE_RULE
```

Comment:
Because neither "style" nor "style_clean" represent actual (filename) artifacts of the make process--and, furthermore, to instruct make that they're both always out of date, so as to run them as requested :P